xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,686 @@
+   </para>
+   
+    <para> 
+        <literal>  
+     send dhcp-client-identifier 1:0:a0:24:ab:fb:9c;
+     </literal>
+  </para>
+  
+  <para>
+     Now install and run <command>lldpd</command>, the Link Layer Discovery Protocol daemon. You will use this to verify correct connectivity between your compute nodes.
+ </para>
+ 
+ <screen><prompt>root #  </prompt><command>zypper in lldpd</command> 
+<prompt>root #  </prompt><command>systemctl start lldpd.service</command> 
+<prompt>root #  </prompt><command>lldpctl</command> </screen>
+
+<para>
+    Give it a few minutes to scan its LLDP neighbors. When the scan is completed make a note of the <literal>PortDescr:</literal> line for the Ethernet interface(s) on the ACI fabric. It should look like this example, with the nodes and port switch detailed:
+</para>   
+  
+<para>
+    <literal>
+    PortDescr: topology/pod-1/paths-102/pathep-[eth1/34]
+    </literal>
+</para> 
+
+<warning>
+    <title>Do not proceed if you do not see proper output from lldpctl</title>
+    <para>If your compute nodes do not see each other in the LLDP scan, go back and check your network settings.
+        </para>
+</warning>
+</sect1>
+
+<sect1 xml:id="sec.openvswitch">
+<title>Install Openvswitch on Control and Compute Nodes</title>
+<para>
+    Install Open vSwitch on your control and compute nodes, version 2.4.1 or greater. The first command installs the necessary packages, and the second command verifies the version number.
+</para>
+<screen><prompt>root # </prompt><command>zypper in openvswitch openvswitch-switch openvswitch-kmp-default</command>
+<prompt>root # </prompt><command>modinfo openvswitch</command>
+</screen>
+
+<para>
+    Create the required OVS (open virtual switch) bridges. Run these commands on the control node:
+</para>
+<screen><prompt>root # </prompt><command>systemctl start openvswitch</command>
+<prompt>root # </prompt><command>systemctl enable openvswitch</command>
+<prompt>root # </prompt><command>ovs-vsctl add-br br-int</command>
+</screen>
+
+<para>
+    On the compute nodes you will also create <literal>vxlan</literal> ports.
+</para>
+
+<screen><prompt>root # </prompt><command>systemctl start openvswitch</command>
+<prompt>root # </prompt><command>systemctl enable openvswitch</command>
+<prompt>root # </prompt><command>ovs-vsctl add-br br-int</command>
+<prompt>root # </prompt><command>ovs-vsctl set bridge br-int protocols=[]</command>
+<prompt>root # </prompt><command>ovs-vsctl add-port br-int br-int_vxlan0 \
+    -- set Interface br-int_vxlan0 type=vxlan options:remote_ip=flow \
+    options:key=flow options:dst_port=8472</command>
+</screen>
```

Comment:
This is a lot of commands. Here you can use sudo -s to open a root shell that allows to enter the commands without having to type "sudo" at the beginning of each line:

    <screen>sudo -s
    systemctl start openvswitch
    ...
    ovs-vsctl add-port ...
    exit</screen>